### DISCUSSION TITLE
provide link to markup in demo page.

### DISCUSSION THREAD

**Discussion by markdav-is** [Megabit/Blazorise#3156] [Ideas]
I'm taking a deep dive into Blazorise today and I'm challenged to find the markup that support the demo.  for example:

demo page: https://bootstrap5demo.blazorise.com/tests/forms
maps to  https://github.com/Megabit/Blazorise/blob/master/Demos/Blazorise.Demo/Pages/Tests/FormsPage.razor

it would be useful to either:
 - provide a page-level link to the code on github for the page
 - publish the razor markup with the demo site and expose it in a code block on the page

see boostrap for example: https://getbootstrap.com/docs/5.0/forms/overview/

thanks for all the great work!

**Comment by stsrki** [comment]
We considered it already but decided to not do it at this time. It would require too much boilerplate to add markup code to the demo. Demo apps are mainly used for a quick showcase of the Blazorise UI capabilities, and most of all for our internal testings and development to make sure all our component's UI is rendered properly.

Real example markup with code samples is reserved for our documentation pages. eg. https://blazorise.com/docs/components/text

Although, we might consider adding a URL to the demo source code on each demo page.

**Reply by markdav-is** [reply]
thanks so much for the link to this:  https://blazorise.com/docs/components/text  that would be really handy on the provider page for sure.  SO much great documentation.  I really appreciate it.
